sdk/core/azure-core-http-netty/src/main/resources/META-INF/native-image/com.azure/azure-core-http-netty/jni-config.json

```diff
@@ -0,0 +1,968 @@
+          "java.lang.String[]",
+          "int"
+        ]
+      },
+      {
+        "name": "setTmpDHLength",
+        "parameterTypes": [
+          "long",
+          "int"
+        ]
+      },
+      {
+        "name": "setSessionIdContext",
+        "parameterTypes": [
+          "long",
+          "byte[]"
+        ]
+      },
+      {
+        "name": "setMode",
+        "parameterTypes": [
+          "long",
+          "int"
+        ]
+      },
+      {
+        "name": "getMode",
+        "parameterTypes": [
+          "long"
+        ]
+      },
+      {
+        "name": "enableOcsp",
+        "parameterTypes": [
+          "long",
+          "boolean"
+        ]
+      },
+      {
+        "name": "disableOcsp",
+        "parameterTypes": [
+          "long"
+        ]
+      },
+      {
+        "name": "getSslCtx",
+        "parameterTypes": [
+          "long"
+        ]
+      },
+      {
+        "name": "setUseTasks",
+        "parameterTypes": [
+          "long",
+          "boolean"
+        ]
+      },
+      {
+        "name": "setPrivateKeyMethod",
+        "parameterTypes": [
+          "long",
+          "io.netty.internal.tcnative.SSLPrivateKeyMethod"
+        ]
+      },
+      {
+        "name": "setSSLSessionCache",
+        "parameterTypes": [
+          "long",
+          "io.netty.internal.tcnative.SSLSessionCache"
+        ]
+      },
+      {
+        "name": "setNumTickets",
+        "parameterTypes": [
+          "long",
+          "int"
+        ]
+      },
+      {
+        "name": "setSessionTicketKeys0",
+        "parameterTypes": [
+          "long",
+          "byte[]"
+        ]
+      }
+    ]
+  },
+  {
+    "name": "io.netty.channel.kqueue.KQueueStaticallyReferencedJniMethods"
+  },
+  {
+    "name": "io.netty.channel.kqueue.Native"
+  },
+  {
+    "name": "io.netty.channel.kqueue.BsdSocket",
+    "methods": [
+      {
+        "name": "sendFile",
+        "parameterTypes": [
+          "int",
+          "io.netty.channel.DefaultFileRegion",
+          "long",
+          "long",
+          "long"
+        ]
+      },
+      {
+        "name": "getAcceptFilter",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "getTcpNoPush",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "getSndLowAt",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "getPeerCredentials",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "setAcceptFilter",
+        "parameterTypes": [
+          "int",
+          "java.lang.String",
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setTcpNoPush",
+        "parameterTypes": [
+          "int",
+          "int"
+        ]
+      },
+      {
+        "name": "setSndLowAt",
+        "parameterTypes": [
+          "int",
+          "int"
+        ]
+      }
+    ]
+  }
+]
```

Comment:
Is there a way nowadays to not require this netty-specific configuration?

Reply:
I can look into this as a follow-up to this PR.